### ISSUE TITLE
Inefficient unblocking in readfish targets

### ISSUE THREAD

**Issue by utrori** [LooseLab/readfish#81]
Hello,

Readfish targets seems to be working as reads from chr21 and chr22 are longer with the selection, but the read length distribution is much longer than what is expected.
Also, the reported mapping speed is unlike the example provided; R value (reads processed?) and times are much larger.
I guess that my hardwares are not bottlenecking the performance since the usage of CPU, GPU and memory are all around 50% during the run.
When I perform unblock-all the read length distribution is just as the example.

MinKnow: 4.0.4
Guppy: 4.0.15
Readfish: 0.0.5a2
GPU: GTX1080
CPU: Ryzen 5 2400g
Memory: 32GB

[ru_test_1.log](https://github.com/LooseLab/readfish/files/5309448/ru_test_1.log)
[hum_chr_selection.toml.txt](https://github.com/LooseLab/readfish/files/5309399/hum_chr_selection.toml.txt)
![Screenshot from 2020-10-01 13-20-22_trimmed](https://user-images.githubusercontent.com/5716037/94768170-9959e200-03e9-11eb-9242-b2993ac759a9.jpg)
![Screenshot from 2020-10-01 13-22-44_trimmed](https://user-images.githubusercontent.com/5716037/94768197-b42c5680-03e9-11eb-8f98-04de28b547ce.jpg)

**Comment by mattloose** [comment]
HI,

OK - my standard response would be to try the fast mode - but you have done this....
So....

Try this....

Find the config file for your fast mode basecaller - likely at /opt/ont/guppy/data/dna_r9.4.1_450bps_fast.cfg

Copy this file and give it a new name - e.g dna_r9.4.1_450bps_even_faster.cfg

Edit this file and comment out the line beginning:

as_model_file

Update your toml file to use this config.

Test away and post your results here.

Matt

**Comment by utrori** [comment]
Hi Matt,

It worked wonderfully.
Thank you very much again!
![Screenshot from 2020-10-07 11-45-12_trimeed](https://user-images.githubusercontent.com/5716037/95284902-6d7fa600-089a-11eb-98b6-2d5795a74ff6.jpg)
[summary_ru.txt](https://github.com/LooseLab/readfish/files/5338141/summary_ru.txt)

**Comment by Freemacl** [comment]
Just to add to this, I tried this fix and it worked for me as well. 

Interestingly, yesterday when I tried this same simulation, it was >0.4 seconds when using dna_r9.4.1_450bps_hac in the toml file, but switching to dna_r9.4.1_450bps_fast.cfg solved that. Today, I found that it was running too slow even with dna_r9.4.1_450bps_fast.cfg in the toml file, so I tried this dna_r9.4.1_450bps_even_faster.cfg approach and it worked.

The only difference between today and yesterday is that I just noticed I have some processes running in the background (compressing large files) that seem to be impacting ReadFish performance. Good to know for future runs!
